helm/reana/README.md

```diff
@@ -68,6 +68,11 @@ This Helm automatically prefixes all names using the release name to avoid colli
 | `components.reana_workflow_engine_snakemake.environment` | [REANA-Workflow-Engine-Snakemake](https://github.com/reanahub/reana-workflow-engine-snakemake) environment variables | `{}`            |
 | `components.reana_workflow_engine_snakemake.image`       | [REANA-Workflow-Engine-Snakemake image](https://hub.docker.com/r/reanahub/reana-workflow-engine-snakemake) to use | `docker.io/reanahub/reana-workflow-engine-snakemake:<chart-release-version>` |
 | `compute_backends`                                       | List of supported compute backends (kubernetes, htcondorcern, slurmcern)             | "kubernetes"                                    |
+| `dask.enabled` | Install dask-kubernetes-operator custom resources in the cluster to support Dask workflows | false |
+| `dask.cluster_max_memory_limit` | Max memory limit for Dask clusters | "16Gi" | 
+| `dask.cluster_default_number_of_workers` | Number of workers in Dask clusters by default | 2 | 
+| `dask.cluster_default_single_worker_memory` | Amount of memory used by a single Dask worker by default | "2Gi" |
+| `dask.cluster_max_single_worker_memory` | Maximum amount of memory that can be used by a single Dask worker | "8Gi" |
```

Comment:
The maximum amount of memory that users can ask for the single Dask worker

Reply:
BTW we may want to chat IRL about these, since the autoscaler could affect the variables and their names and/or documentation strings.